src/proto/parameters.proto

```diff
@@ -210,8 +210,8 @@ message ShootOrPassPlayConfig
         [default = 0.5, (bounds).min_double_value = 0.0, (bounds).max_double_value = 1.0];
 
     // threshold to decide if ball hasn't been kicked
-    required double ball_kicked_threshold = 6
-        [default = 0.5, (bounds).min_double_value = 0.0, (bounds).max_double_value = 5.0];
+    required double ball_not_kicked_threshold = 6
+        [default = 0.3, (bounds).min_double_value = 0.0, (bounds).max_double_value = 5.0];
```

Comment:
We might eventually want to tune this value so we don't accidentally trigger `passFound` when the robot is moving faster than 0.3m/s with the ball.